src/fragments/start/getting-started/reactnative/setup.mdx

```diff
@@ -1,58 +1,209 @@
+## Create a new project
+
 To get started, initialize a new React Native project.
 
+With `expo-cli` you can develop for **Android**, **iOS**, and **web** within minutes.
```

Comment:
The old [Authenticator](https://docs.amplify.aws/ui/auth/authenticator/q/framework/react-native/) has a live link to a new [Authenitcator](https://ui.docs.amplify.aws/react/connected-components/authenticator). The docs are live and connected components are working on both native and web.

I thought it would be a good idea to make a tutorial that's gonna deal with both, and make the exact same example as for React.

Reply:
Regarding any links we have in this Getting Started guide, I think any links here pointing to Authenticator should probably point towards the new Authenticator.

Regarding the web, while we are aware some of our UI components and library categories may work on Expo/RN web, we currently don't officially support it and therefore do not want to include the language or steps which imply the support. If, at a future time, we do verify and commit to supporting Expo/RN web we can update the documentations accordingly. At this time, however, I think it best that we remove any references to Expo or RN web. (This is different, of course, from React web)

Reply:
I have removed all hosting and web references. I would keep `Expo CLI` since it is present in the current tutorial, and official RN documentation.